src/app/auth/userlogin/userlogin.component.ts

```diff
@@ -79,50 +59,78 @@ export class UserloginComponent extends BaseComponent implements OnInit {
-                                            data.agency.indexOf('General Services Administration') > -1;
-            const currentUser = new Currentuser(data.firstName, data.lastName, data.agency, data.email, data.position, data.tempPassword);
-
-            this.user.saveUser(currentUser);
-            console.log(currentUser);
-            if (currentUser.tempPassword !== '') {
-              this.router.navigate(['/user/updatePassword']).catch(r => console.log(r));
-            } else {
-              this.router.navigate(['/home']).catch(r => console.log(r));
-            }
-            // this.router.navigate(['/home']);
-          },
-          error: (error) => {
-            this.errorMessage = true;
-            this.errorInformation = error.error.message;
+      this.authService.login(user).subscribe({
+        next: (data) => {
+          console.log(data);
+          localStorage.setItem('token', data.token);
+          localStorage.setItem('firstName', data.firstName);
+          localStorage.setItem('lastName', data.lastName);
+          localStorage.setItem('agency', data.agency);
+          localStorage.setItem('email', data.email);
+          localStorage.setItem('position', data.position);
+          localStorage.setItem('id', data.id);
+          localStorage.setItem('userRole', data.userRole);
+
+          this.authGuard.isLogin = true;
+          this.app.isLogin = true;
+          this.authGuard.isGSAAdmin = (data.userRole === 'Administrator' || data.userRole === 'SRT Program Manager ') &&
+                                      data.agency.indexOf('General Services Administration') > -1;
+          this.app.isGSAAdmin = (data.userRole === 'Administrator' || data.userRole === 'SRT Program Manager ') &&
+                                      data.agency.indexOf('General Services Administration') > -1;
+          const currentUser = new Currentuser(data.firstName, data.lastName, data.agency, data.email, data.position, data.tempPassword);
+
+          this.user.saveUser(currentUser);
+          if (currentUser.tempPassword !== '') {
+            this.router.navigate(['/user/updatePassword']).catch(r => console.log(r));
+          } else {
+            this.router.navigate(['/home']).catch(r => console.log(r));
           }
+        },
+        error: (error) => {
+          this.errorMessage = true;
+          this.errorInformation = error.error.message;
+        }
       });
     }
+  }
 
+  /**
+   * Redirect to the CAS login page for development environment
+   */
+  loginAsDev() {
+    let serverHost = 'localhost:3000';
+    const clientHost = window.location.hostname;
+
+    if (clientHost !== 'localhost') {
+      serverHost = environment.SERVER_URL;
+    }
+
+    window.location.href = `http://${serverHost}/api/casLogin`;
+  }
+
+  /**
+   * Redirect to Login.gov login based on the current environment
+   */
+  loginWithLoginGov() {
+    const clientHost = window.location.hostname;
+    let serverHost = '';
```

Comment:
Look into utilizing the environment variable for SERVER_URL.

Would transform this function into something more simple like:
```
import { environment } from '../environment/environment' (get the proper path)

let serverHost = environment.SERVER_URL

if (serverHost === '') {
      serverHost = 'localhost:3000';
    }

window.location.href = `http://${serverHost}/login`;
```